.github/workflows/advance-zed.yml

```diff
@@ -51,8 +51,10 @@ jobs:
         with:
           name: artifacts-${{ matrix.os }}
           path: |
-            run/playwright-itest
-            !run/**/SS
+            packages/zui-player/run/playwright-itest
+            !packages/zui-player/run/**/SS
```

Comment:
@jameskerr: Do you have a good memory for details about this path exclusion? I found your [comment](https://github.com/brimdata/zui/pull/2030/files#r780560865) from when it got added:

> This got our artifact uploads working again. We ignore the symlinked files that electron created.

However, in my recent runs in CI, whether tests succeed or fail, I'm not seeing files or symlinks created that match that path. If it's possible that it's a remnant from some past behavior that's unlikely to return, I'd be inclined to remove it on the assumption it's dead code.